King-Benx/news-watchtower-5
# material design workflow [Delivers #164682501]

https://www.pivotaltracker.com/story/show/164682501

**Overview**
This PR applies material design guidelines of arrangement of views. This enhances the appeal of the application hence improving UI/UX

**Acceptance Criteria**
- Application should work as usual.
- Layouts should respect material design.

**How to manually test**
- Clone the repository.
- Build the application ```./gradlew build```
- Run the application.
-Observe the presentation.

**Story**

[#164682501]
